lib/publiccloud/utils.pm

```diff
@@ -254,12 +255,19 @@ sub gcloud_install {
     record_info('GCE', script_output('gcloud version'));
 }
 
+sub get_ssh_private_key_path {
+    my $algorithm = '~/.ssh/id_ed25519';
+    $algorithm = '~/.ssh/id_rsa' if (is_azure() || get_var('PUBLIC_CLOUD_SLES4SAP'));
+    return $algorithm;
```

Comment:
The variable name is a bit of a misnomer that we could avoid like this:

```suggestion
    return (is_azure() || get_var('PUBLIC_CLOUD_SLES4SAP') ? '~/.ssh/id_rsa' : '~/.ssh/id_ed25519';
```

Reply:
Done.